examples/tests/test-echo.sh

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
+
+set -e
+
+RELEASE=$1
+
+cargo run $RELEASE --bin fluvio -- topic delete echo || true
```

Comment:
I see that why `cargo run` is used for compilation.
I would prefer that to put in explicit either explicit `cargo build` or in the Makefile to clear that it is for build step

Reply:
Do you mean in the test script use `cargo build` followed by `cargo run`? Or do you mean to take the `cargo build` out of the test script all together?

Reply:
If you look at CI steps, it is split into build and test in order take advantage of SCCACHE.   If you try to build and test in a single step, you may run into timeout issue.   We don't want to have timeout for build because it may vary but running should be bounded